#source/scss/variables.scss

```
@@ -0,0 +1,6 @@
+$color_1: #fb4b4d;
```

Более конкретные и понятные названия давай переменным, к примеру:

```suggestion
$primary_color: #fb4b4d;
$secondary_color: #8b83f6;
$font_size_large: 58px;
$font_size_xlarge: 65px;
$border_thin: 0.5px solid #757575;
$animation_move: move-linear 5s linear infinite alternate;
```